perf/benchmark/configs/mixer_cpu_mem.yaml

```diff
@@ -0,0 +1,19 @@
+# Configuration Set1: CPU and memory with mixer enabled
+mixer_mode: "mixer"
+conn:
+    - 16
+qps:
+    - 10
+    - 100
+    - 500
+    - 1000
+    - 2000
+    - 3000
+duration: 240
+size: 1024
```

Comment:
by default the size is 1024, we can delete this field?

Reply:
I would prefer to keep it here as it would be an example if people want to add more configs based on it